app/views/layouts/application.html.haml

```diff
@@ -39,5 +39,6 @@
     = javascript_include_tag "//maps.google.com/maps/api/js?v=3.24&key=#{Figaro.env.GOOGLE_MAPS_KEY}"
     = javascript_include_tag '//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js'
     = javascript_include_tag '//cdn.rawgit.com/printercu/google-maps-utility-library-v3-read-only/master/infobox/src/infobox_packed.js'
+    = javascript_include_tag '//cdn.rawgit.com/namuol/cheet.js/master/cheet.min.js'
```

Comment:
Uh, is there any way we can not link to an asset on GitHub? Especially one on master that can be changed at any moment?

Does this library exist on cdnjs? If not, can we vendorize this?